### DISCUSSION TITLE
My Link

### DISCUSSION THREAD

**Discussion by Lumosity427** [TrickJoker312/Unblocked-Sites#10] [Ideas]
hey. I see that you put up my link bhij.w3spaces.com, thx! But it did get closed by an admin. Can you replace it with this link instead? https://gmod.w3spaces.com
i also am currently making a gmod game https://schoolfinding.w3spaces.com, but you will be given a KeyCode in-case teachers view it. They keycode is at the very bottem of the website. gives you a link to https://github.com/Lumosity427/KeyCode/tree/main       If you do put this website up, tysm!

**Comment by TrickJoker312** [comment]
Sure! I appreciate you supporting my repository. Thank you!

**Reply by Lumosity427** [reply]
tysm!  I also am working on the most detailed website i have rn, the link is https://blue-square.w3spaces.com/
basically its a site with a home button, games button, widgetbot button, about button, news, and more. I also put a button called dashboard that sends anyone who clicks it to your repository. I hope you put this up!
